src/include/optimizer/physical_operators.h

```diff
@@ -157,8 +158,8 @@ class IndexScan : public OperatorNode<IndexScan> {
    * @return an IndexScan operator
    */
   static Operator Make(catalog::db_oid_t database_oid, catalog::namespace_oid_t namespace_oid,
-                       catalog::index_oid_t index_oid, std::vector<AnnotatedExpression> &&predicates,
-                       std::string table_alias, bool is_for_update,
+                       catalog::index_oid_t index_oid, catalog::table_oid_t table_oid,
```

Comment:
Since the table "owns" the index, the `table_oid_t` should come before `index_oid_t`.